airflow/api_connexion/openapi/v1.yaml

```diff
@@ -3621,7 +3621,10 @@ components:
 
 # The API will provide support for plugins to support various authorization mechanisms.
 # Detailed information will be available in the plugin specification.
-security: []
+security:
```

Comment:
do not change it here, we will add while releasing. ( similar to   https://github.com/apache/airflow-client-python/pull/33)